packages/libs/react-ui/src/components/IconButton/IconButton.css.ts

```diff
@@ -0,0 +1,68 @@
+import { ColorType, sprinkles, vars } from '../../styles';
+
+import {
+  createVar,
+  fallbackVar,
+  globalStyle,
+  style,
+  styleVariants,
+} from '@vanilla-extract/css';
+
+const surfaceColor = createVar(),
+  highContrastColor = createVar();
+
+export const container = style([
+  sprinkles({
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: 'lg',
+    cursor: 'pointer',
+    size: 11,
+  }),
+  {
+    backgroundColor: fallbackVar(surfaceColor, 'transparent'),
+    color: fallbackVar(highContrastColor, vars.colors.neutral5),
+    border: 0,
+    transition: 'opacity .2s ease',
+    selectors: {
+      '&:hover': {
+        opacity: '.6',
+      },
+      '&:focus-visible': {
+        outlineOffset: '2px',
+        outline: '2px solid $$svgColor',
+      },
+    },
+  },
+]);
+
+globalStyle(`${container} > span > svg`, {
+  color: fallbackVar(highContrastColor, vars.colors.neutral5),
+});
```

Comment:
Vanilla Extract doesn't allow to style with selectors from the parent! If styling from the `container` var, the following error message throws:

```
Error: Invalid selector: & > span > svg

Style selectors must target the '&' character (along with any modifiers), e.g. `${parent} &` or `${parent} &:hover`.

This is to ensure that each style block only affects the styling of a single class.

If your selector is targeting another class, you should move it to the style definition for that class, e.g. given we have styles for 'parent' and 'child' elements, instead of adding a selector of `& ${child}`) to 'parent', you should add `${parent} &` to 'child').

If your selector is targeting something global, use the 'globalStyle' function instead, e.g. if you wanted to write `& h1`, you should instead write 'globalStyle(`${parent} h1`, { ... })'
```